fluss-common/src/main/java/com/alibaba/fluss/metadata/DatabaseDescriptor.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (c) 2024 Alibaba Group Holding Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.fluss.metadata;
+
+import com.alibaba.fluss.annotation.PublicEvolving;
+import com.alibaba.fluss.config.ConfigOption;
+import com.alibaba.fluss.config.ConfigurationUtils;
+import com.alibaba.fluss.utils.Preconditions;
+import com.alibaba.fluss.utils.json.DatabaseDescriptorJsonSerde;
+import com.alibaba.fluss.utils.json.JsonSerdeUtils;
+
+import javax.annotation.Nullable;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * Represents the metadata of a database in Fluss.
+ *
+ * <p>It contains all characteristics that can be expressed in a SQL {@code CREATE Database}
+ * statement, such as schema, primary keys, partition keys, bucket keys, and options.
```

Comment:
The sentence `such as schema, primary keys, partition keys, bucket keys, and options.` should be removed..